types/fabric/fabric-impl.d.ts

```diff
@@ -5492,6 +5492,30 @@ interface IAllFilters {
          */
         fromObject(object: any): ITintFilter;
     };
+    Gamma: {
```

Comment:
This is the only option in the entire body that isn't in alphabetic sorted order - probably best to put it in its correct place

Reply:
thank your for the correction, i've push latest commit with sorted alphabetical order, please review